### PR TITLE
l10n: add missing italian transactions

### PR DESCRIPTION
**Pull request by sadespresso**

### CODE REVIEW

**Comment by sadespresso** [comment]
If you have time, it'll be really nice if you can skim thru this and comment @albertorizzi

**Comment by albertorizzi** [comment]
> If you have time, it'll be really nice if you can skim thru this and comment @albertorizzi

I dont't understand, you tell me to check if these translations in the commit are correct?

**Comment by sadespresso** [comment]
Yes :D :D

File: assets/l10n/it_IT.json [hunk 1]
```diff
@@ -71,6 +72,7 @@
     "account.edit": "Modifica conto",
     "account.edit.selectCurrency": "Seleziona una valuta",
     "account.transactions": "Transazioni",
+    "account.transactions.title": "Li transazioni di \"{account}\"",
```
Comment (albertorizzi) [inline assets/l10n/it_IT.json:75]: Correct translation: "Transazioni di \"{account}\""

**Comment by albertorizzi** [comment]
The other translations are OK! Thank you very much! @sadespresso

**Pull request merged**